src/language/_kinds.ts

```diff
@@ -0,0 +1,65 @@
+/** Name */
+export const NAME = 'Name';
+
+/** Document */
+export const DOCUMENT = 'Document';
+export const OPERATION_DEFINITION = 'OperationDefinition';
+export const VARIABLE_DEFINITION = 'VariableDefinition';
```

Comment:
```suggestion
export const DOCUMENT = 'Document';
export type DOCUMENT = typeof DOCUMENT
export const OPERATION_DEFINITION = 'OperationDefinition';
export type OPERATION_DEFINITION = typeof OPERATION_DEFINITION
export const VARIABLE_DEFINITION = 'VariableDefinition';
// ...
```

Reply:
The reason I did not do these is mainly that it would cause a huge pain with our eslint rule no-redeclare

Reply:
Would aslant complain about that? If so though, could we just ignore with an ignore directive at the top of the file?